File: examples/munge/column_slicing/ex-h2o.R
library(h2o)
path <- "data/iris/iris_wheader.csv"
h2o.init()
df <- h2o.importFile(path)

# slice 1 column by index
c1 <- df[,1]

# slice 1 column by name
c1_1 <- df[, "sepal_len"]

# slice cols by vector of indexes
cols <- df[, 1:4]

# slice cols by vector of names
cols_1 <- df[, c("sepal_len", "sepal_wid", "petal_len", "petal_wid")]

File: examples/munge/column_slicing/ex-h2o.py
import h2o 
h2o.init()
path = "data/iris/iris_wheader.csv"
df = h2o.import_file(path=path)

# slice 1 column by index
c1 = df[:,0]

# slice 1 column by name
c1_1 = df[:, "sepal_len"]

# slice cols by list of indexes
cols = df[:, range(4)]

# slice cols by a list of names
cols_1 = df[:, ["sepal_len", "sepal_wid", "petal_len", "petal_wid"]]
